package main.java.tasks.units;

public class TestException extends Exception {

    public TestException(){
        super("The result of test is wrong");
    }
}
